# Request
string filename
---
# Response
bool success
string message
